.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Release
+on:
+  release:
+    types:
+      # works for building pre-releases
+      - published
+      # works for releases and changing from pre-release to release
+      - released
+
+env: 
+ 
+  DOCKER_IMAGE: newrelic/nri-kube-events
+  RELEASE_VERSION: ${{ github.event.release.tag_name }}
+  S3_PATH: s3://nr-downloads-main/infrastructure_agent
+  PRERELEASE: ${{ github.event.release.prerelease }}
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract version from release tag
+        run: |
+          echo "RELEASE_VERSION=$(echo $RELEASE_VERSION | grep -oE '([0-9.]*[0-9])$')" >> $GITHUB_ENV
+
+      - name: Test for prerelase
+        if: ${{ github.event.release.prerelease }}
+        run: |
+          echo "RELEASE_VERSION=$RELEASE_VERSION-pre" >> $GITHUB_ENV
+          echo "S3_PATH=$S3_PATH/test" >> $GITHUB_ENV
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and push
+        env:
+          DOCKER_IMAGE_NAME: ${{ env.DOCKER_IMAGE }}:${{ env.RELEASE_VERSION }}
+        run: |
+          make docker-build
+          echo "docker push $DOCKER_IMAGE_NAME"
```

Comment:
Leaving a comment here so we don't forget to remove the `echo` when getting this live.

Reply:
nice catch. Forgot to remove that